src/Inkluzitron/Program.cs

```diff
@@ -128,8 +135,11 @@ static private async Task MainAsync(string[] args)
             provider.GetRequiredService<ReactionsModule>();
             provider.GetRequiredService<PointsService>();
 
-            var runtime = provider.GetRequiredService<RuntimeService>();
-            await runtime.StartAsync();
+            var runtimeService = provider.GetRequiredService<RuntimeService>();
+            await runtimeService.StartAsync();
+
+            foreach (var startable in provider.GetServices<ILifecycleControl>())
```

Comment:
Možná bych na toto využil HostedServices (z nugetu knihovnu) https://docs.microsoft.com/cs-cz/aspnet/core/fundamentals/host/hosted-services?view=aspnetcore-5.0&tabs=visual-studio

Reply:
Naplánované úlohy jsou ale jediná věc, která by tohle v aktuální situaci použila. Trochu jsem to ořezal a zůstal z toho jen `IRuntimeEventHandler`. Souhlasím, že pokud by bylo větší využití, tak by bylo lepší použít HostedServices než to bastlit sám.